raft/diff_test.go

```diff
@@ -55,7 +55,7 @@ func mustTemp(pre, body string) string {
 }
 
 func ltoa(l *raftLog) string {
-	s := fmt.Sprintf("committed: %d\n", l.committed)
+	s := fmt.Sprintf("lastIndex: %d\n", l.lastIndex())
```

Comment:
Why do you change this?

Reply:
I am curious why this PR is much bigger than https://github.com/etcd-io/etcd/pull/14411, since they are almost following the same logic ? I am not sure whether it's related to this change?

Reply:
The various tests that are using this really wanted to see what's in the raft log. That this was called "committed" was a misnomer, the tests really didn't care whether it was committed or not. It just so happened that append==commit in the single-node case prior to this change.